mixin.scss

```diff
@@ -0,0 +1,32 @@
+@function bk-calculateTypeOffset($line-height, $font-size, $descdender-height-scale){
+  $line-height-scale: $line-height / $font-size;
+  $line-height-scale: (($line-height-scale - 1) / 2) + $descdender-height-scale;
```

Comment:
Rather than mutate the `$line-height-scale` variable here, you can just return this line right?